src/controllers/index.js

```diff
@@ -1,5 +1,12 @@
-import defaultMessage from '../utils/index';
+import Logs from '../collections/logs';
+import errorMessage from '../utils/error';
 
 export default function(req, res) {
-    res.send(defaultMessage());
+    new Logs().fetch().then((log) => {
+        console.log(`log: ${log}`);
+        res.send(log);
+    }).catch((err) => {
+        console.log(`Error fetching logs: '${err}'.`);
+        res.send(errorMessage());
```

Comment:
this can be accomplished by calling next(err); and then we can have a middleware at the end that handles the error. to do this, the function should accept (req, res, next)